cmd/storj-sim/network.go

```diff
@@ -293,7 +330,7 @@ func newNetwork(flags *Flags) (*Processes, error) {
 				"--debug.addr", net.JoinHostPort(host, port(satellitePeer, i, debugPeerHTTP)),
 			},
 		})
-
+		process.WaitForStart(redisServers[i])
```

Comment:
Since it's already waiting for the satellite to start-up, there probably isn't a need to wait for the redis-server to start separately.

Reply:
yep i removed it... i think i may have pushed right as you commented, so there are recent changes